command/command.go

```diff
@@ -51,6 +87,15 @@ func grabAPIKey(c *gcli.Context) string {
 	return apiKey
 }
 
+func grabLogPath() string {
```

Comment:
This function called for every log action to fetch path from configuration, imo we should call it at one in the beginning

Reply:
Done